app/_includes/md/admin-listen.md

```diff
@@ -5,16 +5,16 @@
    > <br>
    > If you need to expose the `admin_listen` port to the internet in a production environment,
    > {% if_version lte:2.8.x
-   %}[secure it with authentication](/gateway/{{ include.kong_version }}/admin-api/secure-admin-api).{% endif_version %}{% if_version gte:3.0.x
-   %}[secure it with authentication](/gateway/{{ include.kong_version }}/production/running-kong/secure-admin-api).{% endif_version %}
+   %}[secure it with authentication](/gateway/{{page.kong_version}}/admin-api/secure-admin-api).{% endif_version %}{% if_version gte:3.0.x
+   %}[secure it with authentication](/gateway/{{page.kong_version}}/production/running-kong/secure-admin-api).{% endif_version %}
```

Comment:
This will break unexpectedly, as the included is pulled in with `include_cached`.

The correct fix is to provide `kong_version=page.kong_version` wherever we do `{% include_cached md/admin-listen.md`

Reply:
Ack - I know this 🤦‍♀️ will fix properly.